database/migrations/2018_08_05_145124_create_places_visiteds_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePlacesVisitedsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('places_visiteds', function (Blueprint $table) {
```

Comment:
Name can be `visited_places` or `user_place_checkin`.

Reply:
let's stay on - `user_place_checkin` ?